changelog.md

```diff
@@ -3,15 +3,49 @@
 ## Current development version (0.2.0-SNAPSHOT)
 
 **New Features**
-*   Path to AAS Environment model can be set in persistence config
-*   Provide File Persistence as additional default implementation of the persistence interface
-*   Add basic for basic username & password authentication for asset connections (MQTT, OPC UA, HTTP)
-*   validation now checks for unsupported datatypes
-*   always print version info when starting via CLI
+*   Persistence
+	*   File-based persistence added
+	*   Each persistence implementation can now be configured to use a given AAS model as initial value
+
+*   Asset Connection
+	*   HTTP asset connection added
+	*   Basic authentication (username & password) added
```

Comment:
"Only" for MQTT, OPCUA and HTTP. Here it sounds like the basic authentication was added to the common asset connection config.